src/components/ProductCard/ProductCard.spec.js

```diff
@@ -0,0 +1,17 @@
+import { render, screen } from '@testing-library/react'
+import ProductCard from './ProductCard'
+
+describe('ProductCard', () => {
+  it('renders product card elements', () => {
+    render(<ProductCard name="productname" description="product description" />)
```

Comment:
Would be nice to test it with real data as you do in your other tests 🙂

Reply:
Changed it, thanks.